packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -30,6 +30,14 @@ export interface BucketDeploymentProps {
    */
   readonly destinationKeyPrefix?: string;
 
+  /**
+   * If this is set to false, files in the destination bucket that
+   * do not exist in the asset, will NOT be deleted during deployment (create/update).
+   *
```

Comment:
Add @see with a link to AWS CLI --delete option

Reply:
👍